python_ta/cfg/graph.py

```diff
@@ -72,11 +52,15 @@ def add_arguments(self, args: Arguments) -> None:
         args.parent.cfg = self
         args.parent.cfg_block = self.start
 
-        if ExprRef is not Any:
+        try:
+            from ..z3.z3_parser import Z3Parser
+
             # Parse types
             parser = Z3Parser()
             z3_vars = parser.parse_arguments(args)
             self.z3_vars.update(z3_vars)
+        except ImportError:
+            return
```

Comment:
The return is good; however, please also use `logging.error` to display an error message to tell the user that even though they tried to enable Z3, the module couldn't be imported.